include/eve/module/core/regular/is_not_equal.hpp

```diff
@@ -24,9 +24,9 @@
 namespace eve
 {
   template<typename Options>
-  struct is_not_equal_t : strict_elementwise_callable<is_not_equal_t, Options, numeric_option, definitely_option>
+  struct is_not_equal_t : logical_elementwise_callable<is_not_equal_t, Options, numeric_option, definitely_option>
   {
-    template<value T, value U>
+    template<typename T, typename U>
```

Comment:
is_not_equal is not a logical element wise callable.
It's just not. 
We need to figure out what's up.